.github/workflows/check_links.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: 'Check links'
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "12 10 * * 2"
```

Comment:
Does the `workflow_dispatch` trigger allow the workflow to be started from the GH UI?

The workflow takes 40-60s to run, which is about the same length of time it takes to build the documentation. Am I right in thinking that when this PR is merged, then building the documentation would take something like 80-120s? If so, then could we _not_ run it on each PR, accepting that a few links might be broken between scheduled runs?

Reply:
Yes, `workflow_dispatch` is if you want to run manually for any reason.

The workflows here should all run concurrently, as far as I can tell, provided the organisation doesn't hit its limit of [concurrent jobs (60)](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits).

That all said: if we want to make it minimally irritating for contributors (potentially getting a failure on a PR via an existing broken link), let's try it just on a schedule first, and then add it on all PRs if we think it's useful.

Reply:
Updated in 4cf5922c69fa520929786a20d0d4b9313bf840ec.